db/migrate/20240314230145_create_platformcredentials.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePlatformcredentials < ActiveRecord::Migration[7.1]
+  def change
+    create_table :platformcredentials do |t|
```

Comment:
```suggestion
    create_table :platform_credentials do |t|
```
SQL convention is to snake_case multi-word table names.